config.def.h

```diff
@@ -87,6 +87,17 @@ static float maxlatency = 33;
  */
 static unsigned int su_timeout = 200;
 
+/*
+ * Autoscroll interval when use mouse to select
+ */
```

Comment:
This is just a suggestion. Would it be more descriptive this way?

```suggestion
/*
 * Specifies how fast the screen scrolls when you select text and drag the
 * mouse to the top or bottom of the screen.
 */
```

Reply:
fix